tox.ini

```diff
@@ -102,7 +104,9 @@ commands = python -m pylint --reports=n --rcfile=.pylintrc {[base]source_paths}
 deps =
     {[testenv]deps}
     -e certbot-ci
-commands = mypy {[base]source_paths} --strict-equality
+commands =
+    mypy {[base]mypy_no_strict_source_paths} --strict-equality
+    mypy {[base]mypy_strict_source_paths} --strict-equality --strict
```

Comment:
nit: i think it might be worth moving `--strict-equality` to `mypy.ini` to remove the duplication here

Reply:
I agree that's a good idea! It is done.